domains/gamingboots.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "A description",
+    "repo": "https://github.com/gamingboots/gamingboots.github.io",
+    "owner": {
+      "username": "gamingboots",
+      "email": "gamngboots19@gmail.com",
+      "twitter": "your-twitter-username"

```

Comment:
this is **NOT** a valid twittter username.

Reply:
lol epic username if you don't have a twitter acc then just delete line 7

Reply:
:mmlol: big brain